### ISSUE TITLE
(Feature Request) Would it be possible to implement the 'Pixel Protect' feature in 'Content Aware Scale'.

### ISSUE THREAD

**Issue by bluep55** [photopea/photopea#5200]
Would it be possible to implement the 'Pixel Protect' feature in 'Content Aware Scale' similar to how it works in Photoshop.
Example shown in this tutorial on YouTube from about 4 minutes in.
Regards.
https://www.youtube.com/watch?v=-nvUuHKkLgk

**Comment by camoody1** [comment]
This would be an outstanding addition to the program.

**Comment by photopea** [comment]
I have added the option to protect pixels according to a certain channel, you can choose the channel at the top menu.

**Comment by camoody1** [comment]
Could you write up a short tutorial on this with a sample image? Or, better yet, record a short video showing how this would work? Thank you

**Comment by photopea** [comment]
It works exactly as in Adobe Photoshop. You can use e.g. this video https://www.youtube.com/watch?v=-nvUuHKkLgk .

Maybe you need to look for how to make channels in Photoshop, too.

**Comment by bluep55** [comment]
It works ok for me horizontally (scaling from left or right side), but I cannot get it to work scaling vertically (scaling from top or bottom).
Regards.

**Comment by photopea** [comment]
Could you show  a screenshot or any other proof? Because it works both horizontally and vertically in my case.

**Comment by bluep55** [comment]
I'm pretty sure I am doing everything correctly, if not I apologise.
I've attached an image taken from the Photopea Gallery which I think is a good example, then used Content Aware Scale using Photopea and Photoshop to compare the finished results. I've attached images showing both results. 
![Screenshot From Photopea Gallery](https://user-images.githubusercontent.com/118609932/212730995-34a74612-97a8-4268-b808-f2a8cf8fca62.jpg)
![Screenshot Photopea](https://user-images.githubusercontent.com/118609932/212731001-16afeb50-de39-4645-bc71-ec349328f3bd.jpg)
![Screenshot Photoshop](https://user-images.githubusercontent.com/118609932/212731007-380abca7-ff56-4a79-9c60-6e755c750605.jpg)
